clappr/clappr-android
# Update Gradle to v3.4.1 and Robolectric to v4.3

## Goal

Update Gradle to version 3.4.1 and Robolectric to version 4.3.

## How to test

- Build the Player: ./gradlew clean build
- Run all the tests with Android Studio
- Create a Snapshot
- Run the sample and check if everything is working fine
